sql/education_teachers(grade1-12).sql

```diff
@@ -0,0 +1,270 @@
+--
+-- PostgreSQL database dump
+--
+
+SET statement_timeout = 0;
+SET lock_timeout = 0;
+SET client_encoding = 'UTF8';
+SET standard_conforming_strings = on;
+SET check_function_bodies = false;
+SET client_min_messages = warning;
+
+SET search_path = public, pg_catalog;
+
+
+ALTER TABLE IF EXISTS ONLY public.education_teachers DROP CONSTRAINT IF EXISTS education_teachers_pkey;
+DROP TABLE IF EXISTS public.education_teachers;
+
+SET search_path = public, pg_catalog;
+
+SET default_tablespace = '';
+
+SET default_with_oids = false;
+
+--
+-- Name: education_teachers; Type: TABLE; Schema: public; Tablespace:
+--
+
+CREATE TABLE education_teachers(
+    geo_level character varying(15) NOT NULL,
+    total integer NOT NULL,
+    "teacher type" character varying(128) NOT NULL,
+    geo_code character varying(10) NOT NULL
+);
+
+
+--
+-- Data for Name: education_teachers; Type: TABLE DATA; Schema: public
+--
+
+
+
+COPY education_teachers( geo_level, total, "teacher type", geo_code) FROM stdin WITH DELIMITER ',';
+country,109109,Approved teachers,NP

```

Comment:
We should be consistent about capitalization of the words in the "teacher type" column, which means "teachers" should be capitalized on all three types or in none of them.